server/src/dto/UserDto.hh

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <oatpp/core/macro/codegen.hpp>
+#include <oatpp/core/Types.hpp>
+
+#include "PageDto.hh"
+
+#include OATPP_CODEGEN_BEGIN(DTO)
+
+namespace server::dto {
+
+    class UserDto : public oatpp::DTO {
+        DTO_INIT(UserDto, DTO)
+
+        DTO_FIELD(UInt64, id, "id");
+
+        DTO_FIELD(String, email, "email");
+
+        DTO_FIELD(String, username, "username");
+
+        DTO_FIELD(String, password, "password");
+
+        DTO_FIELD(Int32, role, "role"); // 0 - admin, 1 - user //TODO: enum
```

Comment:
Should probably be done before merging.

Reply:
fixed in 19466743ef838762aa40d8d8a3bb9af33c8a3dfb